client/css/widgets/spinner.css

```diff
@@ -11,8 +11,9 @@
 
 .spinner div.spinner {
   position: absolute;
-  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 110 110' width='110' height='110'%3E%3Cstyle%3E tspan %7B white-space:pre %7D .shp0 %7B fill: %23232328 %7D .shp1 %7B fill: %23a1d6cc %7D .shp2 %7B fill: %2352bbae %7D %3C/style%3E%3Cpath id='Forma 3' class='shp0' d='M18.67 58.5L18.67 53.5L92 53.5L92 58.5L18.67 58.5ZM83.5 66.72L79.96 63.18L91.52 51.63L95.05 55.17L83.5 66.72ZM80.06 47.72L83.6 44.19L95.15 55.74L91.61 59.27L80.06 47.72Z' /%3E%3Cpath id='Forma 1' class='shp1' d='M55.5 79C42.5 79 32 68.5 32 55.5C32 42.5 42.5 32 55.5 32C68.5 32 79 42.5 79 55.5C79 68.5 68.5 79 55.5 79Z' /%3E%3Cpath id='Camada 2' fill-rule='evenodd' class='shp2' d='M47.64 76.19C39.92 72.41 35.94 64.73 38.73 59.02C41.53 53.31 50.04 51.75 57.76 55.54C57.98 55.65 58.2 55.76 58.41 55.87C52.07 57.95 47.82 62.89 48.4 68.13C48.97 73.15 53.81 76.92 60.07 77.77C56.42 78.74 51.91 78.28 47.64 76.19Z' /%3E%3Cpath id='Camada 2 copy 2' fill-rule='evenodd' class='shp2' d='M40.11 38.84C46.5 33.08 55.1 32.25 59.36 36.97C63.62 41.69 61.9 50.17 55.51 55.93C55.33 56.09 55.14 56.25 54.96 56.4C55.39 49.74 52.39 43.95 47.3 42.55C42.43 41.2 37.13 44.3 34.01 49.79C34.47 46.04 36.58 42.03 40.11 38.84Z' /%3E%3Cpath id='Camada 2 copy 3' fill-rule='evenodd' class='shp2' d='M77.62 49.72C79.89 58.02 76.75 66.08 70.62 67.75C64.48 69.42 57.69 64.07 55.43 55.77C55.37 55.53 55.31 55.3 55.25 55.06C61.01 58.45 67.53 58.38 71.07 54.47C74.47 50.73 74.1 44.6 70.6 39.35C73.74 41.44 76.37 45.13 77.62 49.72Z' /%3E%3Cpath id='Forma 2' class='shp2' d='M55.5 59C53.56 59 52 57.44 52 55.5C52 53.56 53.56 52 55.5 52C57.44 52 59 53.56 59 55.5C59 57.44 57.44 59 55.5 59Z' /%3E%3C/svg%3E");
-  background-size: contain;
+  background-image: url(/i/spinner/SpinnerVTTblue.svg);
+  background-size: 70% 70%;
```

Comment:
Going from `contain` to `70% 70%` seems wrong. Did you test this with different spinner sizes (and aspect ratios)?

Reply:
This is what works with the new file

Reply:
Probably would be better to adjust the image file so that it fits properly with `contain` rather than using a magic number of `70% 70%` in the background size.